contracts/examples/bonding-curve/src/tokens/fungible_token.rs

```diff
@@ -0,0 +1,191 @@
+elrond_wasm::imports!();
+elrond_wasm::derive_imports!();
+
+use crate::{
+	events,
+	function_selector::FunctionSelector,
+	function_selector::SupplyType,
+	function_selector::{CurveArguments, Token},
+	storage,
+};
+
+const TOKEN_NUM_DECIMALS: usize = 0;
```

Comment:
Why use 0 for the decimals? EGLD has 18 decimals for example. So amount of 1e18 = 1 EGLD. I think you should either use a something other than 0 (18 for example), or make it configurable (receive the value as parameter)